### PR TITLE
[#56165] 14.3 Documentation: Unselect projects from the project selector and searchbar in context menu

### PR DESCRIPTION
**Pull request by MayaBerd**
…tor  https://community.openproject.org/work_packages/56165

[#56165] 14.3 Documentation: Unselect projects from the project selector

https://community.openproject.org/work_packages/56165

### CODE REVIEW

**Review by HDinger — APPROVED** [review]
Lgtm 👍

File: docs/user-guide/agile-boards/README.md [hunk 1]
```diff
@@ -8,11 +8,9 @@ keywords: agile boards, Kanban, Scrum, agile project management, action boards
 
 # Boards for Agile Project Management
 
-Boards support agile project management methodologies, such as Scrum or Kanban.
+Boards support agile project management methodologies, such as Scrum or Kanban. OpenProject agile boards enable you to keep track tasks, bugs, features, risks and any other work package types. Boards consist of lists (columns) and cards (work packages). You can choose between a Basic board and various advanced Action boards.
```
Comment (marcalcobe) [inline docs/user-guide/agile-boards/README.md:11]: @MayaBerd I think Agile should always be written in capital "A" as its a methodology name.

File: docs/user-guide/agile-boards/README.md [hunk 2]
```diff
@@ -29,25 +27,30 @@ Our Agile boards can be for anything you would like to keep track of within your
...
+You can also view the boards on the instance level via [the global modules menu](../home/global-modules/#boards). 
 
-You can create as many Agile boards in a project as you need and configure them to your needs. First, you need to create a new Boards view.
+![Boards module in OpenProject](openproject_userguide_boards_overview.png)
 
-If you haven't done so yet, [activate the Boards module](../projects/project-settings/modules) within your project. Also, we recommend to verify [Roles and Permissions](../../system-admin-guide/users-permissions/roles-permissions/) within your system's Administration.
+## Create a new board
 
-Click on the green **+Board** button to create a new Board view.
+You can create as many agile boards in a project as you need and configure them to your needs. First, you need to create a new Board. To do that, click on the green **+Board** button.
```
Comment (marcalcobe) [inline docs/user-guide/agile-boards/README.md:43]: @MayaBerd same here with the capital Agile

**Review by marcalcobe — COMMENTED** [review]
Hi @MayaBerd, sorry for the late review on that. I just added two minor comments but they are not critical at all.

**Comment by MayaBerd** [comment]
> Hi @MayaBerd, sorry for the late review on that. I just added two minor comments but they are not critical at all.

No worries, thanks for your feedback, I'll include it into another PR I am working on.

**Pull request merged**